### ISSUE TITLE
Narrator Scan mode gets stuck at the end of WPF

### ISSUE THREAD

**Issue by SivaGara** [nvaccess/nvda#13819]
Actual result: NVDA not moving to the next element in scan mode
Expected Result: In NVDA scan mode screen reader should be narrate and access the WPF (Windows Application)

**Comment by CyrilleB79** [comment]
Hi @SivaGara 

1. The title of this issue is talking of the Narrator. Do you confirm that you are talking of NVDA instead and not of the Narrator?
2. The issue description is lacking a lot of details. For your issue to be taken into account, you should complete all the fields of the issue template. More specifically, you should indicate the steps to reproduce the issue so that other people may reproduce it and investigate.
3. Try to use NVDA's terms; in NVDA the equivalent of Narrator's scan mode is called browse mode.

Many thanks.

**Comment by seanbudd** [comment]
Welcome @SivaGara ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- Please give clear steps on reproducing the problem using NVDA, so we can reproduce the issue.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers
